InternalDocs/parser.md

```diff
@@ -200,88 +200,88 @@ first alternative, like so:
 
 ### ``( e )`` (grouping operator)
 
-Match ``e``.
+    Match ``e``.
 
 ```
     rule_name: (e)
 ```
 
-A slightly more complex and useful example includes using the grouping
-operator together with the repeat operator:
+    A slightly more complex and useful example includes using the grouping
+    operator together with the repeat operator:
 
 ```
     rule_name: (e1 e2)*
 ```
 
 ### ``[ e ] or e?``
 
-Optionally match ``e``.
+    Optionally match ``e``.
 
 ```
     rule_name: [e]
 ```
 
-A more useful example includes defining that a trailing comma is
-optional:
+    A more useful example includes defining that a trailing comma is
+    optional:
 
 ```
     rule_name: e (',' e)* [',']
 ```
 
 ### ``e*``
 
-Match zero or more occurrences of ``e``.
+    Match zero or more occurrences of ``e``.
 
 ```
     rule_name: (e1 e2)*
 ```
 
 ### ``e+``
 
-Match one or more occurrences of ``e``.
+    Match one or more occurrences of ``e``.
 
 ```
     rule_name: (e1 e2)+
 ```
 
 ### ``s.e+``
 
-Match one or more occurrences of ``e``, separated by ``s``. The generated
-parse tree does not include the separator. This is otherwise identical to
-``(e (s e)*)``.
+    Match one or more occurrences of ``e``, separated by ``s``. The generated
+    parse tree does not include the separator. This is otherwise identical to
+    ``(e (s e)*)``.
 
 ```
     rule_name: ','.e+
 ```
 
 ### ``&e`` (positive lookahead)
 
-Succeed if ``e`` can be parsed, without consuming any input.
+    Succeed if ``e`` can be parsed, without consuming any input.
 
 ### ``!e`` (negative lookahead)
 
-Fail if ``e`` can be parsed, without consuming any input.
+    Fail if ``e`` can be parsed, without consuming any input.
 
-An example taken from the Python grammar specifies that a primary
-consists of an atom, which is not followed by a ``.`` or a ``(`` or a
-``[``:
+    An example taken from the Python grammar specifies that a primary
+    consists of an atom, which is not followed by a ``.`` or a ``(`` or a
+    ``[``:
 
 ```
     primary: atom !'.' !'(' !'['
 ```
 
 ### ``~``
 
-Commit to the current alternative, even if it fails to parse (this is called
-the "cut").
+    Commit to the current alternative, even if it fails to parse (this is called
```

Comment:
Nice @iritkatriel. It may make sense to have a brief description on line 273 so the ``~`` stands out a bit more.

Reply:
Turns out these indents are not rendering in markdown as I hoped they would. Maybe table is better.

Reply:
https://gist.github.com/JacobCoffee/5e577dc91115cd401afebd54d0eb1a0b is sort've how 2 columns would look. I don't like that you lose the large chunks of descriptions like we had in `~` (In this example,...), but also didn't like the blob of text in the cell for description being so large. 😬

Reply:
That's certainly better than what we currently have. I'd remove the " Example:" in each of the boxes.